tests/serialized_test_data_generation/Makefile

```diff
@@ -0,0 +1,75 @@
+GCR_URL = us.gcr.io/vcm-ml
+REGRESSION_DATA_STORAGE_BUCKET=gs://vcm-fv3gfs-serialized-regression-data
+#<some large conceptual version change>.<serialization statement change>.<hotfix>
+FORTRAN_VERSION=7.1.0
+CWD=$(shell pwd)
+EXPERIMENT ?=c12_6ranks_standard
+
+TEST_DATA_HOST ?=$(CWD)/test_data/$(EXPERIMENT)
+RUNDIR_HOST ?=$(CWD)/rundir/$(EXPERIMENT)
+
+FORTRAN_DIR=$(CWD)/../../
+COMPILED_IMAGE ?= $(GCR_URL)/fv3gfs-compiled:$(FORTRAN_VERSION)-serialize
+
+RUNDIR_CONTAINER=/rundir
+TEST_DATA_CONTAINER=$(RUNDIR_CONTAINER)/test_data
+
+TEST_DATA_TARFILE=dat_files.tar.gz
+TEST_DATA_BUCKET=$(REGRESSION_DATA_STORAGE_BUCKET)/$(FORTRAN_VERSION)/$(EXPERIMENT)
+FV3CONFIG_DIR=$(CWD)/external/fv3config
+LOOKUP_BUCKET=$(shell eval gsutil -q stat $(TEST_DATA_BUCKET)/MetaData-Generator_rank0.json || echo 1)
+
+install_deps:
+	pip3 install -r ../../requirements.txt	
+
+setup_rundir: 
```

Comment:
Do we want to run `pip3 install`  to create a run directory? This feels like an unexpected side-effect. If you can, please remove the `install_deps` call and explicitly use `make install_deps setup_rundir` when you need to run both.